ThePortugueseMan.HabitTracker/HabitsLibrary/MainTable.cs

```diff
@@ -0,0 +1,44 @@
+﻿using DataBaseLibrary;
+using Microsoft.Data.Sqlite;
+using System;
+
+namespace HabitsLibrary;
+
+public class MainTable
+{
+    private static DataBaseCommands dbCommands = new();
+    public string? tableName;
+    public string? connectionString;
```

Comment:
Why are these nullable?

Reply:
Visual studio keeps giving me warnings about assigning null values to non nullable types (especially when reading from console), so that was the way I found to shut it up 😁
Should I ignore the warnings or is there a better way to deal with this?

Reply:
@ThePortugueseMan yeah the default warnings are very strict. You can get rid of them: https://stackoverflow.com/questions/73273384/get-rid-of-nullable-warnings